www/static/plugins/searchbar.jsx

```diff
@@ -8,9 +8,9 @@ var SearchBar = createClass({
 
     // polyfill of sorts for string refs
     linkRef: function(component, name) {
-        let cache = component._linkedRefs || (component._linkedRefs = {});
+        var cache = component._linkedRefs || (component._linkedRefs = {});
         if (!component.refs) component.refs = {};
-        return cache[name] || (cache[name] = c => {
+        return cache[name] || (cache[name] = function(c) {
             component.refs[name] = c;
```

Comment:
should this be:
`return component.refs[name] = c;` ? if not the assignment doesn't make sense

Reply:
Or is the cache caching a function? (I'm going by what arrow functions typically do)

Reply:
Ah hah. Good catch. Arrow functions have implicit returns.